istio.VERSION

```diff
@@ -1,9 +1,9 @@
 # DO NOT EDIT THIS FILE MANUALLY instead use
 # tests/updateVersion.sh (see tests/README.md)
 export CA_HUB="docker.io/istio"
-export CA_TAG="2017-05-09-17.26.56"
+export CA_TAG="0.1.0"
 export MIXER_HUB="docker.io/istio"
-export MIXER_TAG="2017-05-09-20.54.56"
-export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/stable-621b2c4/artifacts/istioctl"
+export MIXER_TAG="0.1.0"
+export ISTIOCTL_URL="https://storage.googleapis.com/istio-release/releases/alpha-0.1.0/istioctl"
```

Comment:
I wish I add this information before (I asked multiple time the version of this release). The image are already tagged and the tags are pushed in the repos (auth, manager, mixer). This is just a number we can have the first production release be 0.2.0. We can also change naming scheme after.